frontend/common/src/messages/commonMessages.ts

```diff
@@ -30,6 +30,10 @@ const messages = defineMessages({
     defaultMessage: "Go back to my profile",
     description: "Link text for button to return to user profile",
   },
+  notApplicable: {
```

Comment:
I don't see this key referenced anywhere in the codebase, is there something I'm missing?

Reply:
Oops, I was using this earlier and just forgot to remove it. Removed with [remove unused common message](https://github.com/GCTC-NTGC/gc-digital-talent/pull/3695/commits/e728cc52069b82cb809504802c86be0b3c312e1d)